sdk/cosmos/azure-cosmos/azure/cosmos/http_constants.py

```diff
@@ -193,6 +193,7 @@ class HttpHeaders:
     PartitionKey = "x-ms-documentdb-partitionkey"
     EnableCrossPartitionQuery = "x-ms-documentdb-query-enablecrosspartition"
     PartitionKeyRangeID = "x-ms-documentdb-partitionkeyrangeid"
+    PhysicalPartitionId = "x-ms-cosmos-physical-partition-id"
```

Comment:
where this header is being used?

Reply:
in our diagnostic logs - it was showing up as REDACTED because we didn't have it listed here